src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -1251,6 +1257,9 @@ def _get_log(url, user_name, password, log_file=None):
                 if not data:
                     break
                 f.write(data)
+    elif number_of_lines:  # print last 'n' number of lines
+        api_str = r.read().decode(encoding='utf-8')
+        _print_site_logs(api_str, int(number_of_lines), 512000, http, headers)
```

Comment:
please add comment for why we are limiting to `512000`? I suggest mention it in the help for the argument of `-l`, so people won't be surprised by the truncating